okta/applicationAccessibility.go

```diff
@@ -0,0 +1,30 @@
+/*
+* Copyright 2018 - Present Okta, Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// AUTO-GENERATED!  DO NOT EDIT FILE DIRECTLY
+
+package okta
+
+import (
+)
+
+type ApplicationAccessibility struct {
+	ErrorRedirectUrl string `json:"errorRedirectUrl,omitempty"`
+	LoginRedirectUrl string `json:"loginRedirectUrl,omitempty"`
+	SelfService bool `json:"selfService,omitempty"`
```

Comment:
I don't know how reasonable it is to run `gofmt` on the generated code, but it'd be nice.

Reply:
I will add a task to run gofmt after the `make build` command

Reply:
See #15